### DISCUSSION TITLE
JIRA integration

### DISCUSSION THREAD

**Discussion by cyrus-za** [rollbar/rollbar.js#988] [Ideas]
Not really related to rollbar.js specifically, but there seems to be no specific place to log bugs. I filed a support ticket, but none of the options on there were about technical issues, so I am posting it here too.

We love using the rollbar integrations with slack, jira, email etc. 

There is one annoying issue though. When a rollbar item is resolved and it updates the jira ticket to Done, it does it twice. This means our jira-slack integration spams twice. See below
<img width="691" alt="image" src="https://user-images.githubusercontent.com/1845861/109487023-55e4cc00-7a8c-11eb-8dfc-fb40b37971a9.png">

Can you please do a check before transitioning to a state to see if it's already in the state. Doing  simple GET to jira to get the issue you can do 
```ts
if(issue.fields.status.name === newState) return;
```

**Comment by waltjones** [comment]
@cyrus-za Thank you for taking time to report this. I'll forward it to the team.

**Comment by waltjones** [comment]
CH80756
